editor/import/resource_importer_wav.cpp

```diff
@@ -35,6 +35,9 @@
 #include "core/os/file_access.h"
 #include "scene/resources/audio_stream_sample.h"
 
+const float kTrimDBLimit = -50;
+const int kTrimFadeOutFrames = 500;
```

Comment:
Is there a specific reason for naming those with a `kCamelCase` style, or just personal preference? If the latter, please use `TRIM_DB_LIMIT` and `TRIM_FADE_OUT_FRAMES` instead, as used in the rest of the code base for constants.

Reply:
No reason other than I didn't check the convention for constants in another file first. I'll change it.

Reply:
It's changed now and pushed.